release/charts/charts.go

```diff
@@ -0,0 +1,53 @@
+package charts
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"os/exec"
+	"strings"
+
+	ecmConfig "github.com/rancher/ecm-distro-tools/cmd/release/config"
+)
+
+// ListLifecycleStatus prints the lifecycle status of the charts
+func ListLifecycleStatus(ctx context.Context, c *ecmConfig.ChartsRelease, branch, chart string) error {
+	var branchArg, chartArg string
+
+	branchArg = "--branch-version=" + branch
+	if chart != "" {
+		chartArg = "--chart=" + chart
+	}
+
+	if err := executeChartsBuildScripts(c.Workspace, "lifecycle-status", branchArg, chartArg); err != nil {
+		return err
+	}
+
+	fmt.Printf("generated log files for inspection at: \n%s\n", c.Workspace+"/logs/")
```

Comment:
Let's remove this in favor of printing output from the calling code of this function. Library code shouldn't be putting anything to a log or STDOUT unless in debug.

Reply:
@briandowns I removed the print from here to the parent calling function, please check if this is right now.